frontend/src/api/user.js

```diff
@@ -0,0 +1,28 @@
+export const getUser = async () => {
+  try {
```

Comment:
I am little confused as to why there is duplication here?

Couldn't you use the getUserById and get the token that way?

As the token is added to local storage when a user signs in

Reply:
The get user endpoint is not authenticated while get user by id is.

You could use that call but because we're using this to pre-populate edit profile there's the risk someone edits it.

In chrome dev tools go => application => local storage => http://localhost:3000 and we can type in any value we want.  For anything to do with the user's actual account we should use the id stored in the token.

If we are querying for any user it makes sense to pass that users id.  Anytime we want to know about the logged in user we should use the token.